<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_window</class>
 <widget class="QMainWindow" name="Main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>375</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Main_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="showVideo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openVideo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopVideo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="filename">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
